src/algonauts/models/model_loaders.py

```diff
@@ -9,6 +10,15 @@ def load_vgg16():
     return model, transform_image
 
 
+def load_alexnet(num_classes):
+    model = create_alexnet_softmax(num_classes)
+    optimizer = tf.keras.optimizers.Adam()
+    loss = tf.keras.losses.BinaryCrossentropy()
+    model.compile(optimizer=optimizer, loss=loss, metrics='accuracy')
+    transform_image = image_transforms.transform_alexnet
+    return model, transform_image
+
+
 def load_from_file(model_filename, transform_image):
-    model = tf.keras.models.load_model(model_filename, custom_objects={'F1Score': tfa.metrics.F1Score})
+    model = tf.keras.models.load_model(model_filename)  # , custom_objects={'F1Score': tfa.metrics.F1Score})
```

Comment:
it produces an error if tries to load a model with an unrecognised customisations, so you need to pass it in case you used it to train it

Reply:
I acutally removed F1Score because I cannot install tensorflow-addons on my macbook. There is no version compatible with M1 tensorflow dependencies. So I removed F1Score when running it locally.

I should add it back for the code though.